### DISCUSSION TITLE
cannot find any projects.

### DISCUSSION THREAD

**Discussion by SadSock** [oracle/opengrok#4480] [General]
I configured the opengrok service using the official Docker image, but opengrok cannot find any projects.
As shown in the picture below.

![图片](https://github.com/oracle/opengrok/assets/7643216/b3ed77c3-350f-44e1-b844-ba2fb932eb18)

The command I used to start is like this

![图片](https://github.com/oracle/opengrok/assets/7643216/2ec78496-ca73-4bc4-9ab8-6331511809e2)

The directory structure of my project is as follows.

![图片](https://github.com/oracle/opengrok/assets/7643216/f9410f78-1ea4-4200-b662-074e60cb78b4)

Where is the problem?

**Comment by vladak** [comment]
what is the OpenGrok version ?

**Reply by SadSock** [reply]
1.12.13
